source/ui/beamline/AMXRFDetailedDetectorView.cpp

```diff
@@ -167,6 +172,8 @@ void AMXRFDetailedDetectorView::buildEnergyRangeSpinBoxView()
 {
 	AMSpectrumAndPeriodicTableView::buildEnergyRangeSpinBoxView();
 
+	connect(exportButton_, SIGNAL(clicked()), this, SLOT(onExportButtonClicked()));
```

Comment:
@mButlerDM @helfrij I think it might be better to be where we created the button. In this case it is the function `AMSpectrumAndPeriodicTableView::buildEnergyRangeSpinBoxView();`.

@mButlerDM if `onExportButtonClicked()` must be implemented in the Detailed detector view, you can make it a (pure) virtual slot in `AMSpectrumAndPeriodicTableView`.